wazo_acceptance/helpers/sip_phone.py

```diff
@@ -41,7 +41,7 @@ def _add_last_newline(self, data):
         return data
 
 
-class SIPPhone:
+class SIPPhone():
```

Comment:
```suggestion
class SIPPhone:
```
Not sure why the parentheses were added/kept?